dbms/src/Flash/Mpp/ExchangeReceiver.cpp

```diff
@@ -766,6 +824,14 @@ void ExchangeReceiverBase<RPCContext>::connectionDone(
         finishAllMsgChannels();
 }
 
+template <typename RPCContext>
+void ExchangeReceiverBase<RPCContext>::connectionLocalDone()
+{
+    std::lock_guard lock(mu);
+    is_local_conn_alive = false;
```

Comment:
I think we need to consider that in the furture, there may be more than 1 local tunnel in one ExchangeReceiver

Reply:
> I think we need to consider that in the furture, there may be more than 1 local tunnel in one ExchangeReceiver

ok, I will fix it.

Reply:
Never mind, it is reasonable to assume that for each exchange receiver, only here is only one local tunnel.